src/components/icon.js

```diff
@@ -0,0 +1,40 @@
+(() => ({
+  name: 'Icon',
+  icon: 'ContainerIcon',
+  category: 'LAYOUT',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: [],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const { Icons } = window.MaterialUI;
+    const isDev = B.env === 'dev';
+    const { icon } = options;
+    const IconComponent = React.createElement(Icons[icon], {
```

Comment:
This will break, because default value is `None` which does not exist as a Material Icon.
You could either render something else instead of an icon, or remove the `None` option and set an existing icon as default. Why would anyone drag an icon on the canvas and then not pick an icon?